package.json

```diff
@@ -16,7 +16,7 @@
   "author": "Traceur Authors",
   "license": "Apache License 2.0",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=0.12"
```

Comment:
As I understand it we are now compatible with .10 and .12 (and potentially iojs). Is there are reason to force users on to .10?